### PR TITLE
Create angr.io.xml

### PR DESCRIPTION
**Pull request by TotalCaesar659**

### CODE REVIEW

File: src/chrome/content/rules/angr.io.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!--
+        Invalid certificate:
+                www.angr.io
+        Time out:
+                api.angr.io
+                ci.angr.io
+-->
+<ruleset name="angr.io">
+  <target host="angr.io" />
```
Comment (J0WI) [inline src/chrome/content/rules/angr.io.xml]: `docs.angr.io`

File: src/chrome/content/rules/angr.io.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
```
Comment (pipboy96) [inline src/chrome/content/rules/angr.io.xml]: Also add:

```
  <target host="www.angr.io" />
```

and

```
  <rule from="^http://www\.angr\.io/" to="https://angr.io/" />
```

File: src/chrome/content/rules/angr.io.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  <target host="www.angr.io" />
+  <target host="docs.angr.io" />
+
+  <rule from="^http://www\.angr\.io/" to="https://angr.io/" />
```
Comment (pipboy96) [inline src/chrome/content/rules/angr.io.xml]: Please add

```
  <rule from="^http:" to="https:" />
```

after this rule.

Comment (TotalCaesar659) [reply src/chrome/content/rules/angr.io.xml]: Ah, sorry.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**